### PR TITLE
creating preprod and prod environments for sentence-planning

### PR DESCRIPTION
**Pull request by nickmcmahon01**
creating preprod and prod environments for sentence-planning.
creating prod and notprod environments for sentence-planning-auth.

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/oasys-keycloak-development/resources/variables.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
+variable "domain" {
+  default = "oasys-auth-development.service.justice.gov.uk"
```
Comment (vijay-veeranki-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/oasys-keycloak-development/resources/variables.tf]: There are multiple domains in the PR.

You can create one like `oasys-auth.service.justice.gov.uk` and avoid creating multiple domains.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/oasys-keycloak-development/05-certificate.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: oasys-keycloak-development.service.justice.gov.uk
```
Comment (vijay-veeranki-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/oasys-keycloak-development/05-certificate.yaml]: If you manage with one route-53 zone, you can manage your ingress and cert with some thing like
`development.oasys-keycloak.service.justice.gov.uk`

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-development/resources/variables.tf [hunk 3]
```diff
@@ -0,0 +1,35 @@
+variable "domain" {
+  default = "sentence-planning-development.service.justice.gov.uk"
```
Comment (vijay-veeranki-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-development/resources/variables.tf]: Same in here, you don't need multiple zones, can create one with "sentence-planning.service.justice.gov.uk"

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-preprod/05-certificate.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: sentence-planning-preprod.service.justice.gov.uk
```
Comment (vijay-veeranki-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-preprod/05-certificate.yaml]: if you have one zone, you can add this cert and ingress as `preprod.sentence-planning.service.justice.gov.uk

**Comment by vijay-veeranki-moj** [comment]
Hi @nickmcmahon01 , Can you please raise a separate PR for a namespace.

It will be easy for us to review it and also the pipeline can be quicker with separate PR's.

**Comment by nickmcmahon01** [comment]
Thanks @vijay-veeranki-moj, I will split the PR into two and do the subdomain thing.

**Comment by nickmcmahon01** [comment]
@vijay-veeranki-moj I have made these changes now, please look at PR #1754 too.

**Comment by nickmcmahon01** [comment]
Hi Vijay, is this ok to merge now, we have made the requested changes.

**Pull request merged**